### ISSUE TITLE
Old English 

### ISSUE THREAD

**Issue by AnyankaT** [ldtteam/stylecolonies#43]
**Old English** hearkens back to the era of Shakespeare, and is designed around multi-purpose "complexes" which combine work, residence, and guard towers in a single building. It's designed in various woods (spruce, oak, dark oak, and birch) at the first two levels, which are then replaced with stripped woods and finally paper extra framed in dark oak at level 5, with a few buildings also using roan bricks. 

- It's been available on [Planet Minecraft](https://www.planetminecraft.com/mod/old-english-stylepack-for-minecolonies-1-20/) since April and has had nearly 1000 downloads since then. 
- The most recent version (attached) has gotten rid of any third-party mod dependencies. 
- Everything has been updated to follow the required progression - some of the child huts don't strictly follow, but their parent huts make up for it. 
- I'm actively maintaining the style, and adding further deco and alternate huts.
- Because it consists of parent/child huts, it's not really a beginner style, but (at least I think) it's fairly self-explanatory, and I have tried to be responsive to issues and suggestions.

![2024-04-06_10 28 01](https://github.com/user-attachments/assets/61beae61-bf51-4f01-bd63-706966942b0c)

Each complex has a moderate footprint, and the entire colony can be comfortably built in around a 20x20 chunk space.

![2024-09-06_13 31 44](https://github.com/user-attachments/assets/e16d3751-7fe0-474d-83ec-cc1c4e5b648c)
Above is my current colony on our community server - all the buildings are at level 3, with additional roads and bridges I plan to add to the pack when they are ready.

![2024-09-06_11 46 48](https://github.com/user-attachments/assets/e29d344f-da01-459a-b24d-c91e58175f46)
![2024-09-06_13 13 07](https://github.com/user-attachments/assets/fedacdf1-38a8-4687-b7ca-3c3d5314fbf0)
The five levels for the Town Hall, with an interior shot of the built-in restaurant at level 5.

![2024-09-06_11 49 22](https://github.com/user-attachments/assets/ac7915f9-793a-4d2f-b478-7851f3ac77f7)
The five levels for the University. Interior images of the library, school, and residence are below.
![2024-09-06_13 15 26](https://github.com/user-attachments/assets/542b2bdf-1616-434b-98e9-a9b6be5e3860)
![2024-09-06_13 15 42](https://github.com/user-attachments/assets/994c8974-4815-4c8a-9677-3eac8fbc0ac3)
![2024-09-06_13 15 52](https://github.com/user-attachments/assets/3b1c1468-381b-4693-86ca-b85b900e5da1)

The level 5 warehouse (below) includes ten courier offices - with signs over the door indicating their level - and enough residence space upstairs for all couriers.

![2024-09-06_13 20 57](https://github.com/user-attachments/assets/1f13c6cd-e89d-4fc9-9e23-111aed204756)
![2024-09-06_13 19 16](https://github.com/user-attachments/assets/b687c580-d686-494a-87a5-5ad5b0b0a6d7)
![2024-09-06_13 19 42](https://github.com/user-attachments/assets/5f518069-ffdd-42c5-95ce-f53929e3c572)
![2024-09-06_13 20 31](https://github.com/user-attachments/assets/94d4c153-b640-4b58-ada6-434118f36c0b)
![2024-09-06_13 17 58](https://github.com/user-attachments/assets/b6fdac41-997c-42e7-9852-f096f81868e3)
![2024-09-06_13 18 04](https://github.com/user-attachments/assets/a0837fe0-bd61-469d-8b61-8ee61c650f71)

The plantation is based on a 3x3 grid, with the main hut in the middle, surrounded by fields and related buildings such as the apiary and the flower shop.

![plantation](https://github.com/user-attachments/assets/8e609c28-ac65-4e01-a050-ba086c4d9506)

The cookery is included! 

![2024-09-06_11 52 37](https://github.com/user-attachments/assets/37a60a7d-5016-4619-bfb8-6ed6141eef54)
![2024-09-06_13 07 08](https://github.com/user-attachments/assets/5926fbff-9f65-412d-81c3-49b989e114e3)

The crowning jewel of the style is the Cathedral, which includes graveyard, hospital and enchanter, the alchemist in the dome, as well as two residences and guards. A dedicated builder is located in the churchyard, and a ground plan is provided to properly align the building parts.

![2024-09-06_12 12 06](https://github.com/user-attachments/assets/e2b8d2b2-5a41-4f09-ab24-f3cb0d1c8b21)
![2024-09-06_13 09 10](https://github.com/user-attachments/assets/9b583e1e-09cf-438c-a2e6-ba389d193927)
![2024-09-06_13 11 20](https://github.com/user-attachments/assets/6d8bbb35-ff51-4151-a43a-e5597b959304)
![2024-09-06_13 11 35](https://github.com/user-attachments/assets/dc2be339-71e3-46af-832c-aa8a9971b4da)
![2024-09-06_13 07 42](https://github.com/user-attachments/assets/9d26f7e5-dbc9-425d-b327-9f444375cbf7)
![2024-09-06_13 08 30](https://github.com/user-attachments/assets/7ee56b4d-d81c-4df6-8431-294d2c4bd79c)

Newest version of the pack, which includes a few minor changes since I updated PMC earlier today:
[Old English for StyleColonies.zip](https://github.com/user-attachments/files/16907275/Old.English.for.StyleColonies.zip)

I welcome feedback and suggestions for improvements! Thanks for considering the style!

**Comment by Raycoms** [comment]
This looks very nice, do you also have a test/schematic world you can send us, or does it even exist on the schematic server?

**Comment by AnyankaT** [comment]
It's not on the schematic server, because when I developed it I was using Amendments and Supplementaries, so I have my own. I'm glad to give whitelist access per request, though. I just really didn't want to have to reprogram all the command blocks!

Modpack for my style server:  [MC Style Work.zip](https://github.com/user-attachments/files/16907364/MC.Style.Work.zip)

**Comment by Raycoms** [comment]
Some feedback:

I think visually the style is very appealing and looks very neat,

It does go quite overboard on the child buildings indeed though.
There is a huge amount of housing in pretty much all buildings at an almost 0 cost.

The buildings themselves are generally okay cost wise, but come with incredibly cheap and plenty integrated buildings.

Some of the buildings look quite difficult and not very straightforward to assemble. It would probably be easier of the child buildings came already in the main schematic then. Especially considering that the main building will look cut off if that is not the case.

**Comment by AnyankaT** [comment]
Yeah, most of them do place the child hut as part of the first level of the parent. I think the only big exception is the combination of glassblower and mechanics hut.

So should I make the residence parts of the complexes more expensive in terms of material cost? Keep in mind that they're all invisible and can't be accessed directly in survival, so I was trying to calculate their cost into the overall building cost.

**Comment by kezmodius** [comment]
Ray, you said, "Some of the buildings look quite difficult and not very straightforward to assemble." 
Can you remember which ones you thought this about?

I have been beta testing this style and can give the feedback (so far) that any nested hut blocks that I have needed to place are obvious in how to orient the hut block by lining up existing features or having a sign telling you how to place. The child huts are placed by the builder and do not need player assistance. There is also a detailed guide (like we use for Underwater or Cavern) linked in an instruction book in the supply camp. AnyankaT has done well to ensure the players can figure it out in and out of game.
